src/components/Steps.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import React from 'react';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+import { Col, Row, Container } from 'reactstrap';
+
+const Steps = styled.div`
+  background-color: ${props => props.theme.colors.helFog};
+  padding-top: 4em;
+`;
+
+const StepContainer = styled(Container)`
+  width: 37.5em;
+`;
+
+const StepIcon = styled(Col)`
+  text-align: center;
+  margin-bottom: 0.625em;
+`;
+
+const Circle = styled.div`
+  width: 3.125em;
+  height: 3.125em;
+  display: inline-block;
+  border: 0.125em solid #000;
+  border-radius: 50%;
+  padding-top: 0.7em;
+  font-size: 1.2em;
+  color: #fff;
+  margin-bottom: 1em;
+
+  &.active {
```

Comment:
Prefer `props` over classNames, that's why styled-component is for

Reply:
Yes, this will be refactored next when each step will be its own component.